mgo.go

```diff
@@ -383,6 +385,9 @@ func getMongod() (string, error) {
 	if path := os.Getenv("JUJU_MONGOD"); path != "" {
 		paths = append(paths, path)
 	}
+	if featureflag.Enabled(feature.MongoDbSnap) {
```

Comment:
No need for feature flag check. This is an upstream repo and must not bring in any juju/juju deps.
We are just adding a new addition to the search path.